jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Cipher.java

```diff
@@ -19,311 +19,312 @@
 package org.eclipse.jetty.http2;
 
 import java.util.Comparator;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
-import org.eclipse.jetty.util.Index;
+import org.eclipse.jetty.util.StringUtil;
 
 public class HTTP2Cipher
```

Comment:
Looking at the changes made to this class, two things pop into my mind:
 - Do those maps need to be case-insensitive?
 - Aren't there other classes that make use of Index that should instead use a Map?

Reply:
I think yes, unless we can find something that specifies that protocol and ciphernames are specific cases.

There are probably other uses of Index that could be a map... I will look...